### PR TITLE
235 semver changelog

### PR DESCRIPTION
**Pull request by tobybatch**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,17 @@ We provide a set of docker images for the [Kimai v2](https://github.com/kevinpap
 
 The built images are available from [Kimai v2](https://hub.docker.com/repository/docker/kimai/kimai2) at Docker Hub.
 
+## Deving and Contributing
+
+We use commit liniting to generate commits that we can auto generate changelogs from. To set these up you will need node/nvm installed:
```
Comment (jokay) [inline README.md]: Typo: ... lin**~~i~~**ting ...

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+##  (2021-04-29)
+
+
+### Features
+
+* added contrib rules ([ec60151](https://github.com/tobybatch/kimai2/commits/ec60151f86c7da710dd051d3f236e54cfcd2912a))
+* added linting to git commits ([3542601](https://github.com/tobybatch/kimai2/commits/3542601fd87d09a06c3b91ba32a3860f0e0550ef))
+* added nvmrc ([e3e53eb](https://github.com/tobybatch/kimai2/commits/e3e53eb3fc3db9239203da16d9fd811a840902aa))
+* addeed test script ([c24f649](https://github.com/tobybatch/kimai2/commits/c24f64981f09997e28d4dc026f132c7ba4dfbd44))
```
Comment (jokay) [inline CHANGELOG.md]: I think this needs to be removed?

**Review by jokay — CHANGES_REQUESTED** [review]

**Comment by jokay** [comment]
With this change you may think of migrating from [dependabot](https://github.com/tobybatch/kimai2/blob/main/.github/dependabot.yml) to [renovate](https://github.com/apps/renovate) for your repo(s) so it will pin and update the versions in the `package.json` as well.

File: CHANGELOG.md (hunk 2 above)
Comment (tobybatch) [reply CHANGELOG.md]: It's auto generated from the (old bad) commits. I'll try and figure out how to exclude it.

**Review by jokay — APPROVED** [review]

**Pull request merged**